src/MenuGroup/MenuItem.js

```diff
@@ -0,0 +1,122 @@
+// @flow
+
+import * as React from 'react';
+import { View, Platform } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import Separator from './Separator';
+import { Icon } from '../Icon';
+import { StyleSheet, type StylePropType } from '../PlatformStyleSheet';
+import { Text } from '../Text';
+import { Touchable } from '../Touchable';
+
+type Props = {|
+  +title: React.Node,
+  +onPress: () => void,
+
+  +titleStyle?: StylePropType,
+  +icon?: React.Element<typeof Icon>,
+  +subTitle?: React.Node,
+  +subTitleStyle?: StylePropType,
+  +label?: React.Node,
+  +labelStyle?: StylePropType,
+  +hideActionIcon?: boolean,
+|};
+
+export default function MenuItem({
+  title,
+  onPress,
+  icon,
+  titleStyle,
+  subTitle,
+  subTitleStyle,
+  label,
+  labelStyle,
+  hideActionIcon = false,
+}: Props) {
+  const menuIcon =
+    icon != null
+      ? React.cloneElement(icon, {
+          color: icon.props.color ?? defaultTokens.paletteInkLight,
+          size: icon.props.size ?? 'medium',
+          style: [styles.icon, icon.props.style],
+        })
+      : null;
+  return (
+    <View style={styles.container}>
+      <Touchable onPress={onPress} style={styles.wrapper}>
+        <View style={styles.row}>
+          {menuIcon}
+          <View>
+            {label != null && (
+              <Text type="secondary" size="small" style={labelStyle}>
+                {label}
+              </Text>
+            )}
+            <Text type="attention" size="large" style={titleStyle}>
+              {title}
+            </Text>
+            {subTitle != null && (
+              <Text type="secondary" size="small" style={subTitleStyle}>
+                {subTitle}
+              </Text>
+            )}
+          </View>
+          {Platform.OS === 'ios' && hideActionIcon === false && (
+            <View style={styles.actionIcon}>
+              <Icon
+                name="chevron-right"
+                color={defaultTokens.paletteProductNormal}
+              />
+            </View>
+          )}
+        </View>
+      </Touchable>
+      <Separator style={icon == null ? styles.noIcon : styles.iconSeparator} />
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    marginTop: StyleSheet.hairlineWidth,
+    ios: {
+      marginBottom: -StyleSheet.hairlineWidth,
+    },
+  },
+  wrapper: {
+    backgroundColor: defaultTokens.paletteWhite,
+    ios: {
+      paddingTop: 8.5,
+      paddingBottom: 11.5,
+    },
+    android: {
+      paddingTop: 6,
+      paddingBottom: 8.5,
+    },
+
+    paddingStart: 12,
+    paddingEnd: 16,
+  },
+  noIcon: {
+    ios: {
+      marginStart: 12,
+    },
+  },
+  iconSeparator: {
+    ios: {
+      marginStart: 12 + 24 + 12, // wrapperMarginStart + iconWidth + iconMargin
```

Comment:
Maybe create variables at the top and use them in these different places to make this comment redundant. WDYT?

Reply:
I think that sounds like a marvellous idea